readme.md

```diff
@@ -21,12 +21,14 @@
 - [Recipes](https://github.com/avajs/ava/tree/master/docs/recipes)
 - [Testing React Native apps with AVA](https://shift.infinite.red/testing-the-bejeezus-out-of-react-native-apps-with-ava-330f51f8f6c3)
 - [Getting Started with Create React App and AVA](https://semaphoreci.com/community/tutorials/getting-started-with-create-react-app-and-ava)
+- [Effortless unit testing with AVA](https://wecodetheweb.com/2016/04/19/effortless-unit-testing-with-ava/)
 
 
 ## Videos
 
 - [JavaScript Air episode with the AVA team](http://jsair.io/ava)
 - [AVA Casts](http://avacasts.com) - Short screencasts about AVA.
+- [AVA Tutorial](https://www.youtube.com/watch?v=RxLW6-3dk5A) - AVA Tutorial Youtube Link.
```

Comment:
This title too generic. The video look to be more about testing React components with AVA.

Reply:
The description also needs to be improved. And `Youtube` => `YouTube`.

Reply:
Thanks @sindresorhus . So you want me to update the description of youtube link. This is my first pull request, so I am bit of struggling here.

Reply:
Specifically, `AVA Tutorial` is too generic, and `AVA Tutorial Youtube Link.` is not a good description.

Reply:
Thanks for reply @sindresorhus . I will make the changes in pr.